packages/desktop-client/src/components/util/AmountInput.tsx

```diff
@@ -74,8 +74,9 @@ export function AmountInput({
   }
 
   function onInputTextChange(val) {
-    setValue(val ? val : '');
-    onChangeValue?.(val);
+    const value = format(val, 'string').replace(',', '.');
```

Comment:
We should probably do this in onBlur/onUpdate instead of onChange

Reply:
Value has already no information about commas, so we cannot tell if it is for example `42` or `4,2` or `0,42` here in [TransactionEdit.jsx](https://github.com/actualbudget/actual/blob/master/packages/desktop-client/src/components/mobile/transactions/TransactionEdit.jsx#L328). Please correct me if I'm wrong.